### PR TITLE
migrate code from googleapis/java-video-intelligence

### PR DESCRIPTION
**Pull request by anguillanneuf**
- samples: scaffold pom.xml files (#100)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.3.0 (#106)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.13 (#108)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.14 (#114)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.4.0 (#115)
- chore(deps): update dependency com.google.cloud:libraries-bom to v4.4.1 (#118)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.15 (#117)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5 (#129)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.16 (#134)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.17 (#137)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.2.0 (#140)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.3.0 (#146)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.4.0 (#154)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.6.0 (#168)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.18 (#180)
- chore(deps): update dependency com.google.cloud:libraries-bom to v5.7.0 (#181)
- chore(deps): update dependency com.google.cloud:libraries-bom to v6 (#194)
- chore(deps): update dependency com.google.cloud:libraries-bom to v7 (#196)
- chore(deps): update dependency com.google.cloud:libraries-bom to v7.0.1 (#203)
- chore(deps): update dependency com.google.cloud:libraries-bom to v8 (#207)
- chore(deps): update dependency com.google.cloud:libraries-bom to v8.1.0 (#216)
- samples: Add video v1p1beta samples for face detection and video transcription (#1070)
- samples: Update samples due to changes (#1083)
- samples: Add automatic punctuation to video speech transcription (#1165)
- samples: Video Intelligence region tag update (#1183)
- samples: Added beta samples for video object tracking/text detection (#1237)
- samples: Add beta samples for Video Streaming (#1353)
- samples: Flaky test fix (#1490)
- samples: Add new beta samples for Video Intelligence Streaming with an AutoML … (#1568)
- samples: feat: face and person detection samples (#2066)
- samples: video: move samples out of branch (#2300)
- samples: video: fix flaky tests (#2376)
- samples: update shared config (#2443)
- samples: video: update .mp4 file used in test (#2550)
- samples: samples: fix test (#2872)
- samples: samples: fix flaky video stream and text Detection tests (#3438)
- chore(deps): update dependency com.google.cloud:libraries-bom to v9 (#228)
- chore(deps): update dependency com.google.cloud:libraries-bom to v10
- chore(deps): update dependency com.google.cloud:libraries-bom to v11 (#260)
- chore(deps): update dependency com.google.cloud:libraries-bom to v11.1.0 (#266)
- chore(deps): update dependency com.google.cloud:libraries-bom to v12 (#270)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.21 (#261)
- test(deps): update dependency junit:junit to v4.13.1
- chore(deps): update dependency com.google.cloud:libraries-bom to v12.1.0 (#282)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13 (#291)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.1.0 (#297)
- samples: moving beta samples from java-docs and refactored GA samples (#299)
- test(deps): update dependency com.google.truth:truth to v1.1 (#292)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.2.0 (#307)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.3.0 (#309)
- chore(deps): update dependency com.google.cloud:libraries-bom to v13.4.0 (#314)
- chore(deps): update dependency com.google.cloud:libraries-bom to v15 (#323)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16 (#335)
- samples: update person detection to GA (#344)
- fix: incorrect and misleading attribute description for person detection samples
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.2.0 (#363)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.2.1 (#368)
- test(deps): update dependency com.google.truth:truth to v1.1.2 (#380)
- chore(deps): update dependency com.google.cloud:libraries-bom to v16.4.0 (#379)
- docs: generate sample code in the Java microgenerator (#395)
- test(deps): update dependency junit:junit to v4.13.2 (#398)
- chore(deps): update dependency com.google.cloud:libraries-bom to v17 (#408)
- chore: removed wrong comment and updated according to python samples (#409)
- samples: Initial commit of sample
- samples: Update to published version of client library.
- samples: Adds quickstart and test for Video
- samples: updating to latest google-cloud-* dependencies (#723)
- samples: Upgrades client to 0.20.1 and addresses changes to long running operations
- samples: updating video intelligence sample to 0.21.1-alpha (#812)
- samples: Video Intelligence samples (#849)
- samples: Change request type on certain inspection type (#857)
- samples: Auto-update dependencies. (#912)
- samples: Update samples to v1 (#935)
- samples: Updates for storage, translate, unittests, and video. (#994)
- samples: Add video v1p1beta samples for face detection and video transcription (#1070)
- samples: Correct video detect region tags. (#1107)
- samples: Video Intelligence region tag update (#1183)
- samples: Fix broken sample and add test for sample (#1216)
- samples: feat: video speech transcription (#1264)
- samples: Video Intelligence : Object tracking and Text detection (OCR) - GA (#1347)
- samples: Flaky test fix (#1490)
- samples: Adding GA samples for Logo detection (#2393)
- samples: update shared config (#2443)
- samples: video: update .mp4 file used in test (#2550)
- samples: Change video test to not depend on specific inference
- chore(deps): update dependency com.google.cloud:libraries-bom to v18 (#414)
- chore(deps): update dependency com.google.cloud:libraries-bom to v18.1.0 (#427)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19 (#430)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19.1.0 (#441)
- chore(deps): update dependency com.google.cloud:libraries-bom to v19.2.1 (#443)
- samples: updates face and people to v1 (#449)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.22 (#452)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20 (#462)
- chore: removed try/catch & added exceptions at method signature (#456)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.1.0 (#471)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.2.0 (#492)
- chore: removed all catch clauses which were causing error stack trace not appearing in the test result (#464)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.3.0 (#501)
- deps: update dependency com.google.cloud:google-cloud-core to v1.94.8 (#493)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.4.0 (#524)
- chore: increase timeouts for tracking objects samples (#531)
- samples: add streaming_automl_object_tracking sample (#502)
- test(deps): update dependency com.google.truth:truth to v1.1.3 (#537)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.5.0 (#535)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.6.0 (#554)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.0.23 (#553)
- deps: update dependency com.google.cloud:google-cloud-core to v1.95.1 (#550)
- deps: update dependency com.google.cloud:google-cloud-core to v1.95.2 (#559)
- chore: added error message for capturing error (#561)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.7.0 (#574)
- deps: update dependency com.google.cloud:google-cloud-core to v1.95.4 (#573)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.8.0 (#582)
- chore(deps): update dependency com.google.cloud:libraries-bom to v20.9.0 (#594)
- deps: update dependency com.google.cloud:google-cloud-core to v2 (#606)
- deps: update dependency com.google.cloud:google-cloud-core to v2.0.3 (#618)
- deps: update dependency com.google.cloud:google-cloud-core to v2.0.5 (#622)
- chore(deps): update dependency com.google.cloud:libraries-bom to v21 (#630)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.0 (#642)
- chore(deps): update dependency com.google.cloud:libraries-bom to v22 (#654)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.2 (#650)
- chore: migrate to owlbot (#664)
- chore(deps): update dependency com.google.cloud:libraries-bom to v23 (#666)
- Feat: Add code sample for video streaming action recognition.  (#667)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.3 (#671)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.4 (#680)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.6 (#687)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.7 (#696)
- chore(deps): update dependency com.google.cloud:libraries-bom to v23.1.0 (#706)
- deps: update dependency com.google.cloud:google-cloud-core to v2.1.8 (#717)
- deps: update dependency com.google.cloud:google-cloud-core to v2.2.0 (#720)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24 (#729)
- test: Added retries to DetectLogoGcsTest (#730)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.0 (#739)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.1 (#741)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.2 (#748)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.3 (#750)
- chore(deps): update dependency com.google.cloud.samples:shared-configuration to v1.2.0 (#747)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.0 (#761)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.1 (#763)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.1.2 (#768)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.4 (#767)
- deps: update dependency com.google.cloud:google-cloud-core to v2.3.5 (#772)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.2.0 (#784)
- docs: Added retry to DetectIT sample test (#785)
- deps: update dependency com.google.cloud:google-cloud-core to v2.4.0 (#793)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.3.0 (#805)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.0 (#799)
- test: Add retries to streaming shot change detection test (#809)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.1 (#806)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.3 (#812)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.4 (#820)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.5 (#833)
- chore(deps): update dependency com.google.cloud:libraries-bom to v24.4.0 (#840)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.6 (#838)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.8 (#847)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.9 (#851)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25 (#858)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.10 (#857)
- deps: update dependency com.google.cloud:google-cloud-core to v2.5.11 (#866)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.1.0 (#874)
- deps: update dependency com.google.cloud:google-cloud-core to v2.6.1 (#882)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.2.0 (#898)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.3.0 (#905)
- deps: update dependency com.google.cloud:google-cloud-core to v2.7.1 (#906)
- chore(deps): update dependency com.google.cloud:libraries-bom to v25.4.0 (#915)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.0 (#918)
- feat: Enable REST transport for most of Java and Go clients (#920)
- fix: update gapic-generator-java with mock service generation fixes (#922)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.1 (#923)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26 (#935)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.3 (#943)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.4 (#944)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.5 (#945)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.6 (#950)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.7 (#951)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.8 (#953)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.0 (#961)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.9 (#964)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.10 (#965)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.1 (#966)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.11 (#973)
- chore(bazel): Update WORKSPACE files for rules_gapic, gax_java, generator_java versions (#971)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.12 (#979)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.2 (#986)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.13 (#991)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.14 (#993)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.15 (#995)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.16 (#1001)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.17 (#1005)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.18 (#1029)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.19 (#1033)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.20 (#1038)
- chore(deps): update dependency com.google.cloud:libraries-bom to v26.1.3 (#1046)
- deps: update dependency com.google.cloud:google-cloud-core to v2.8.22 (#1050)

Fixes #issue

> It's a good idea to open an issue first for discussion.

- [ ] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [ ] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: video/README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
-# Video Intelligence Samples have been moved
+# [Cloud Video Intelligence: Java Samples](https://github.com/googleapis/java-video-intelligence)
```
Comment (averikitsch) [inline video/README.md:1]: Fix repo url

File: video/pom.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
```
Comment (Shabirmean) [inline video/pom.xml:4]: Can we update this to `com.example.video`

File: video/pom.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  <artifactId>videointelligence-snippets</artifactId>
+  <packaging>jar</packaging>
+  <name>Google Cloud Video Intelligence Snippets</name>
+  <url>https://github.com/googleapis/java-video-intelligence</url>
```
Comment (Shabirmean) [inline video/pom.xml:8]: Can we update this to: `https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/video`

**Review by Shabirmean — CHANGES_REQUESTED** [review]

File: video/pom.xml (hunk 2 above)
Comment (anguillanneuf) [reply video/pom.xml:4]: @Shabirmean this looks fine to me as it makes it clear that it's a google cloud sample, any reason for this change?

File: video/pom.xml (hunk 3 above)
Comment (anguillanneuf) [reply video/pom.xml]: good catch. fixed.

File: video/pom.xml (hunk 2 above)
Comment (Shabirmean) [reply video/pom.xml:4]: This was something we discussed in the migration team chat and agreed to use `com.google.com`. 

**context:** [link](https://chat.google.com/room/AAAA6duiSb8/2iEmMYQxCLg)
**cc:** @averikitsch for more insight

File: video/README.md (hunk 1 above)
Comment (anguillanneuf) [reply video/README.md]: Good catch. done!

**Review by Shabirmean — APPROVED** [review]

**Review by davidcavazos — APPROVED** [review]

**Pull request merged**